VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitVideoFragment.java

```diff
@@ -281,7 +281,7 @@ public void onPositiveClicked() {
                         }
                         @Override
                         public void onNegativeClicked() {
-                            ((VideoListActivity) getActivity()).showInfoMessage(getString(R.string.wifi_off_message));
+                            ((CourseUnitNavigationActivity) getActivity()).showInfoMessage(getString(R.string.wifi_off_message));
```

Comment:
Currently, only the `showInfoMessage()` method seems to be called from the casts which is defined in `BaseFragmentActivity`, so I guess we can safely cast to that instead (as all our `Activity` classes should inherit from it).